editor/src/clj/editor/ui.clj

```diff
@@ -615,8 +615,11 @@
     root))
 
 (defn- load-xml [path]
-  (with-open [stream (io/input-stream (io/resource path))]
-    (xml/parse stream)))
+  (try
+    (with-open [stream (io/input-stream (io/resource path))]
+      (xml/parse stream))
+    (catch Exception e
+      (nil))))
```

Comment:
load_svg_path is accepting nil as a return value and the svg will not be rendered